clients/tabby-chat-panel/src/index.ts

```diff
@@ -69,6 +71,10 @@ export interface ClientApi {
   onCopy: (content: string) => void
 
   onKeyboardEvent: (type: 'keydown' | 'keyup' | 'keypress', event: KeyboardEventInit) => void
+
+  // this is inner function cover by tabby-threads
+  // the function doesn't need to expose to client but can call by client
+  hasCapability?: (capability: ClientApiMethods) => Promise<boolean>
 }
```

Comment:
I think using interface extends could be better?
```
export interface ClientApiMethods {
  // ...
}
export interface ClientApi extends ClientApiMethods {
  hasCapability(method: key of ClientApiMethods): Promise<boolean>
}
```

Reply:
That makes a lot of sense, thanks for the advice. :^